tests/src/test/java/alluxio/server/tieredstore/LostStorageIntegrationTest.java

```diff
@@ -38,6 +53,9 @@
 /**
  * Tests for getting worker lost storage information.
  */
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({StorageDir.class})
```

Comment:
is this because `StorageDir` is `final`?
In case yes, lets change `public final class StorageDir` to be `public class StorageDir`.
In general, we should avoid using `PowerMock` which has a lot limitation